images/linux/scripts/installers/cmake.sh

```diff
@@ -9,7 +9,8 @@ echo "Checking to see if the installer script has already been run"
 if command -v cmake; then
```

Comment:
Can we rid of the if else statement if we would like to install always the latest verison?

Reply:
Good point. If an old CMake version is installed, we need to remove it first I think.

@maxim-lobanov Could you give some context about how likely it it we encounter a preinstalled CMake version? Are the images build from scratch or are the existing images updated (overwritten)?

Reply:
We always build an image from scratch.

Reply:
Is it harmful to just leave it there? Or maybe throw an error/warning instead of just a log message?